cpp/Tooling.md

```diff
@@ -2,6 +2,18 @@
 
 ## C++ Compiler
 
+### C++ Standard
+
+The C++ standard officially supported at BBP is C++14.
+An source project can support C++11 if this is a requirement for its community.
+BBP internal deployment toolchain supports C++17. Therefore software dedicated to BB5 or BlueBrain workstations can pick this standard.
+
+### Minimum Supported Compilers
```

Comment:
This sounds a little dangerous. Does this mean that all code should be tested with GCC5.4 to ensure support for that? As in, will all your compilation flags used in Robert work with all these compilers (and newer ones)?

I Iike
```suggestion
### Compiler Versions Supporting C++14
```

Reply:
IMO we should support GCC 5.4 for:
* open-source projects
* projects deployed on external super computers because as @pramodk dictated, we cannot expect to have the same toolchain than BB5.

Reply:
> As in, will all your compilation flags used in Robert work with all these compilers (and newer ones)?

I should clarify that the compilation flags mentioned below are the ones recommended for development, not deployment.

Reply:
> because as @pramodk dictated 

thank you @tristan0x! 😊 

Matthias - it’s same as Python version. We are not testing/deploying with Python 3.6 version all the time but as developers we should make sure that code can be built with compiler supporting C++14.